Lib/test/test_tools/test_pathfix.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from test.test_tools import import_tool
+
+
+class TestPathFix(unittest.TestCase):
+
+    def setUp(self):
+        self.pathfix = import_tool('pathfix')
+    test_cases = [
```

Comment:
Indentation problem?

Reply:
No, the list of test cases is a class attribute.